tests/e2e/e2e_common.go

```diff
@@ -223,6 +223,25 @@ const (
 		"/api-tokens/authorize"
 )
 
+// test suite labels
```

Comment:
we can var shorten names, something trivial like
slow = "slow"

also mulling whether we need this section or not